doc/contribute/guidelines.rst

```diff
@@ -536,12 +536,21 @@ reference manuals, etc.
 Coding Style
 ============
 
+.. note::
+   Coding style is enforced on any new or modified code, but contributors are
+   not expected to correct the style on existing code that they are not
+   modifying.
+
 .. _Linux kernel coding style:
    https://kernel.org/doc/html/latest/process/coding-style.html
 
+.. _snake case:
+   https://en.wikipedia.org/wiki/Snake_case
+
 In general, follow the `Linux kernel coding style`_, with the following
-exceptions:
+exceptions and clarifications:
 
+* Use `snake case`_ for code and screaming snake case for macros and constants.
```

Comment:
Do people really know what "screaming" means in this context? Why not use all-caps or similar? Perhaps adding two examples could help?

Reply:
It was in the wikipedia link, but I clarified it now.